### ISSUE TITLE
bug: Showing available patches in app info screen instead of patch selection screen

### ISSUE THREAD

**Issue by oSumAtrIX** [ReVanced/revanced-manager#1996]
### Bug description

While this would make sense in the app info screen outside of the "Patch an app" process,  in this process the selected patches vs available patches info should be shown in the patch selector screen here:

![image](https://github.com/ReVanced/revanced-manager/assets/13122796/57fe1df3-3d84-4779-b103-e033272c147b)

and not here:

![image](https://github.com/ReVanced/revanced-manager/assets/13122796/0c914b3e-e2be-4a14-8996-f00e7c2457eb)

### Version of ReVanced Manager and version & name of app you are patching

-

### Installation method

None

### ReVanced Manager logs

```shell
-
```

### Acknowledgements

- [X] This issue is not a duplicate of an existing bug report.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.
- [X] The bug is only related to ReVanced Manager

**Comment by CnC-Robert** [comment]
So selecting an app should take you directly to the patch selector screen?

**Comment by oSumAtrIX** [comment]
No, to the overview is fine. This issue is about the showing the available patches in the incorrect screeb
